acoular/sdinput.py

```diff
@@ -49,6 +49,11 @@ def __init__(self, *args, **kwargs):
     #: Sampling frequency of the signal, changes with sinusdevices
     sample_freq = Property(desc='sampling frequency')
 
+    _sample_freq = Int(0)
+
+    #: Datatype (resolution) of the signal, used as `dtype` in a sd `Stream` object
+    data_type = String('float32', desc='data type (resolution) of signal')
```

Comment:
This should be an Enum of possible values and should also allow certain numpy.dtype value (see sd docs)

Reply:
Resolved in https://github.com/acoular/acoular/pull/285/commits/ec664a66f6f7ac2200f26f2d35283bc9418d92e0 as proposed